client/components/ProjectOptions.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components/native';
+
+
+const ProjectOptions = () => {
+    const ProjectOptionsView = styled.View`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 23px 10px
+    `
+
+    const ProjectOptionsText = styled.Text`
+        font-size: 18px
+    `
+
+    const ProjectOptionsTouch = styled.TouchableOpacity``
+
```

Comment:
these styled components should be outside of the component definition as they need to be static